packages/plugin-core/src/commands/OpenLink.ts

```diff
@@ -25,47 +26,56 @@ export class OpenLinkCommand extends BasicCommand<CommandOpts, CommandOutput> {
   async execute() {
     const ctx = DENDRON_COMMANDS.OPEN_LINK;
     this.L.info({ ctx });
-    if (!isAnythingSelected()) {
-      const error = DendronError.createFromStatus({
-        status: ERROR_STATUS.INVALID_STATE,
-        message: `nothing selected`,
-      });
-      this.L.error({ error });
-      return { error };
-    }
-    const { text } = VSCodeUtils.getSelection();
-    if (_.isUndefined(text)) {
+
+    let text = "";
+
+    text = getURLAt(VSCodeUtils.getActiveTextEditor());
+
+    if (_.isUndefined(text) || text === "") {
       const error = DendronError.createFromStatus({
         status: ERROR_STATUS.INVALID_STATE,
-        message: `nothing selected`,
+        message: `no valid path or URL selected`,
       });
       this.L.error({ error });
       return { error };
     }
-    const wsRoot = getDWorkspace().wsRoot;
     let assetPath: string;
-    if (text.startsWith("asset")) {
-      const vault = PickerUtilsV2.getOrPromptVaultForOpenEditor();
-      assetPath = path.join(vault2Path({ vault, wsRoot }), text);
+    if (
+      text.indexOf(":/") !== -1 ||
+      text.indexOf("/") === 0 ||
+      text.indexOf(":\\") !== -1
```

Comment:
There are a few other path formats that may be good to check for (like "~" or "%APPDATA%" but this should be enough for now.

Reply:
agreed, we can deal with those in separate issues as they come up